src/main/java/de/slikey/effectlib/util/BaseImageEffect.java

```diff
@@ -62,6 +62,11 @@
      * Should it orient to face the player's direction?
      */
     public boolean orient = true;
+    
+    /**
+     * Should it face in the same direction as the location. Obeying yaw and pitch?
+     */
+    public boolean mimic = false;
```

Comment:
Would you mind changing the variable name to `orientPitch`, please? I think that will be easier to understand and more consistent.

Otherwise this is a great add, thank you!